chapter5.Rmd

```diff
@@ -527,56 +623,134 @@ success_msg("Great! Not only is the `subset()` function more concise, it is prob
 
 
 --- type:NormalExercise xp:100 skills:1 key:6a6fe74d3917c37380f7ac616ce084aa7814fb8a
-## Sorting
+## Sorting (1)
 
 Making and creating rankings is one of mankind's favorite affairs. These rankings can be useful (best universities in the world), entertaining (most influential movie stars) or pointless (best 007 look-a-like).
 
-In data analysis you can sort your data according to a certain variable in the data set. In R, this is done with the help of the function [`order()`](http://www.rdocumentation.org/packages/base/functions/order). 
+In data analysis it is often helpful to see your data sorted from smallest to largest, or largest to smallest. In R, this is done with the help of two functions: [`sort()`](http://www.rdocumentation.org/packages/base/functions/sort) and [`order()`](http://www.rdocumentation.org/packages/base/functions/order). 
+
+[`sort()`](http://www.rdocumentation.org/packages/base/functions/sort) is the easiest to understand, so let's consider it first. If you pass a vector to sort, it returns the values from smallest to largest.
+
+```{r}
+a <- c(100, 10, 1000)
+sort(a)
+## [1]   10  100 1000
+```
+
+*** =instructions
+
+`diameter` is shown in your script.
+
+* Use `sort()` to sort the planet diameters from smallest to largest.
```

Comment:
I normally don't use bullets unless there are two or more. In this case, you can just do 

> `diameter` is shown in your script. Use `sort()` to sort the planet diameters from smallest to largest.